MyToDo/ToDoList/ToDoListView.swift

```diff
@@ -13,13 +13,20 @@ struct ToDoListView: View {
 
     var body: some View {
         NavigationView {
-            List {
-                ForEach(viewModel.todos) { todo in
-                    Label(todo.title ?? "", systemImage: todo.category?.imageName ?? "")
-                }.onDelete(perform: viewModel.delete)
+            VStack {
+                SearchBarView() { text in
```

Comment:
```suggestion
                SearchBarView { text in
```

It would look a little more at home in SwiftUI like that (and AFAIK should compile, I didn't check)